From aa6a1face105fcd33932687fb704171e6fb97b25 Mon Sep 17 00:00:00 2001
From: NelloKudo <marshnelloosu@gmail.com>
Date: Wed, 12 Nov 2025 20:59:31 +0100
Subject: [PATCH 1/2] redist tarball

---
 Makefile.in | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/Makefile.in b/Makefile.in
index 43325a57..29aadf57 100644
--- a/Makefile.in
+++ b/Makefile.in
@@ -1419,7 +1419,10 @@ deploy: all
 redist: all
 	mkdir -p $(REDIST_DIR)
 	rsync --delete -arx $(DST_BASE)/ $(REDIST_DIR)
-
+	mv $(REDIST_DIR) $(BUILD_NAME)
+	XZ_OPT="-9 -T$(J)" tar -cvJf $(BUILD_NAME).tar.xz $(BUILD_NAME)
+	sha512sum $(BUILD_NAME).tar.xz > $(BUILD_NAME).sha512sum
+	@echo "Proton build available at $(BUILD_NAME).tar.xz"
 
 ##
 ## make module=$dllname module[32,64]
-- 
2.51.2

