diff --git a/Makefile.in b/Makefile.in
index a0dda579..f66dce15 100644
--- a/Makefile.in
+++ b/Makefile.in
@@ -400,6 +400,8 @@ $(eval $(call rules-makedep,steamexe,i386))
 $(eval $(call rules-makedep,steamexe,x86_64))
 
 
+ifneq ($(WITHOUT_TTS),1)
+
 ##
 ## piper
 ##
@@ -500,6 +502,8 @@ $(eval $(call rules-source,vosk,$(SRCDIR)/vosk-api))
 $(eval $(call rules-cmake,vosk,i386,unix))
 $(eval $(call rules-cmake,vosk,x86_64,unix))
 
+endif # WITHOUT_TTS
+
 
 ##
 ## wine
@@ -544,8 +548,12 @@ WINE_x86_64_AUTOCONF_ARGS += \
   VKD3D_PE_CFLAGS="-I$(VULKAN_HEADERS_x86_64_DST)/include -I$(VKD3D_x86_64_DST)/include/vkd3d" \
   VKD3D_PE_LIBS="-L$(VKD3D_x86_64_LIBDIR)/vkd3d/x86_64-windows -l:libvkd3d-1.dll -l:libvkd3d-shader-1.dll"
 
+ifneq ($(WITHOUT_TTS),1)
 WINE_DEPENDS = gst_orc gstreamer gst_base vkd3d ffmpeg openfst kaldi vosk
 WINE_x86_64_DEPENDS = piper
+else
+WINE_DEPENDS = gst_orc gstreamer gst_base vkd3d ffmpeg
+endif
 
 WINE_i386_LIBDIR = $(WINE_i386_DST)/lib
 WINE_x86_64_LIBDIR = $(WINE_x86_64_DST)/lib
diff --git a/configure.sh b/configure.sh
index 3aebf19c..8c928272 100755
--- a/configure.sh
+++ b/configure.sh
@@ -183,6 +183,9 @@ function configure() {
     if [[ -n "$arg_enable_bear" ]]; then
       echo "ENABLE_BEAR := 1"
     fi
+    if [[ -n "$arg_without_tts" ]]; then
+      echo "WITHOUT_TTS := 1"
+    fi
 
     # Include base
     echo ""
@@ -204,6 +207,7 @@ arg_docker_opts=""
 arg_relabel_volumes=""
 arg_enable_ccache=""
 arg_enable_bear=""
+arg_without_tts=""
 arg_help=""
 invalid_args=""
 function parse_args() {
@@ -252,6 +256,8 @@ function parse_args() {
       arg_enable_ccache="1"
     elif [[ $arg = --enable-bear ]]; then
       arg_enable_bear="1"
+    elif [[ $arg = --without-tts ]]; then
+      arg_without_tts="1"
     elif [[ $arg = --proton-sdk-image ]]; then
       val_used=1
       arg_protonsdk_image="$val"
@@ -309,6 +315,8 @@ usage() {
   "$1" ""
   "$1" "    --enable-bear Invokes make via bear creating compile_commands.json."
   "$1" ""
+  "$1" "    --without-tts Disables text-to-speech libraries (OpenFST, VOSK, Kaldi and Piper)"
+  "$1" ""
   "$1" "  Steam Runtime"
   "$1" "    Proton builds that are to be installed & run under the steam client must be built with"
   "$1" "    the Steam Runtime SDK to ensure compatibility.  See README.md for more information."
-- 
2.51.2

